person.rb

```diff
@@ -20,6 +21,12 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    return if @rentals.include?(rental)
+
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Here passing `rental ` to a function not b able to create rental, I think the proper code should be this. Kindly add it.
```ruby
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Reply:
Well resolved 👍🏽